package.json

```diff
@@ -128,6 +127,7 @@
     "fast-glob": "^3.2.11",
     "fs-teardown": "^0.1.3",
     "glob": "^10.0.0",
+    "globals": "^14.0.0",
```

Comment:
Why do we need it as a dev dependency? Is it being used in some tests?

Reply:
Yes, in tests, for example: 

https://github.com/eslint/eslint/blob/d8068ec70fac050e900dc400510a4ad673e17633/tests/lib/rules/no-global-assign.js#L14

https://github.com/eslint/eslint/blob/d8068ec70fac050e900dc400510a4ad673e17633/tests/lib/rules/no-global-assign.js#L71-L79

Also in `eslint.config.js`:

https://github.com/eslint/eslint/blob/d8068ec70fac050e900dc400510a4ad673e17633/eslint.config.js#L32

https://github.com/eslint/eslint/blob/d8068ec70fac050e900dc400510a4ad673e17633/eslint.config.js#L156-L160